core/backend/src/test/assets/TestGeneratedClasses.ecschema.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ECSchema schemaName="TestGeneratedClasses" alias="tgc" version="1.0.0" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.1">
```

Comment:
it's worth noting that relationships mapped to link tables can have navigation properties on them (pointing to an element and maybe model)

Reply:
I am going to make a note of this but defer supporting that (it currently isn't supported), pending additional work that is in progress.